templates/service/build.sbt

```diff
@@ -0,0 +1,28 @@
+import TestPhases.oneForkedJvmPerTest
+import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, defaultSettings, scalaSettings}
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+
+name := "$!APP_NAME!$"
+
+lazy val root = (project in file("."))
+  .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin)
+  .configs(IntegrationTest)
+  .settings(inConfig(IntegrationTest)(Defaults.itSettings): _*)
+
+scalaSettings
```

Comment:
`scalaSettings` and `defaultSettings()` not needed, already added by `SbtAutoBuildPlugin`
Yeah, I do realise there are there currently :(